src/util/Logger.java

```diff
@@ -0,0 +1,44 @@
+package util;

```

Comment:
You probably know this already but in case you were unaware, there's already a logger in Java (java.util.logging)

Reply:
I am a massive idiot